docs/api/model_selection/importances.rst

```diff
@@ -107,10 +107,54 @@ Taking the mean of the importances may be undesirable for several reasons. For e
     X, y = data.data, data.target
 
     model = LogisticRegression(multi_class="auto", solver="liblinear")
-    viz = FeatureImportances(model, stack=True, relative=False)
+    viz = FeatureImportances(model, stack=True, relative=False, topn=3)
```

Comment:
What was your motivation to add this parameter to the stacked chart docs?

Reply:
That's a mistake from the editing process! Will eliminate.